examples/map.html

```diff
@@ -3,9 +3,9 @@
 <head>
 <meta charset="utf-8" />
 
-<script src="http://cdn.robotwebtools.org/threejs/current/three.js"></script>
-<script src="http://cdn.robotwebtools.org/EventEmitter2/current/eventemitter2.min.js"></script>
-<script src="http://cdn.robotwebtools.org/roslibjs/current/roslib.js"></script>
+<script src="http://static.robotwebtools.org/threejs/current/three.js"></script>
```

Comment:
these changes should really go in a separate PR - the CDN has already migrated, and we should reflect that in our code ASAP

Reply:
It has been already requested to @rctoris. And it doesn't matter as long as it doesn't go in to master branch.

Reply:
I'm not sure I understand what you're saying. The new URLs (with static in them) already work. 

Why don't we make a separate pull request to update them in examples and Readme files right now, while we're still discussing the move to ThreeJS r88?

Reply:
What I am saying is that our stable branch is `master` where we should keep the working code. I am moving on. We need to create a PR to fix some links example anyway when CDN is updated.